### ISSUE TITLE
Richtext within Merge Cells works fine in 5.5 but not in 5.7

### ISSUE THREAD

**Issue by blindmeis** [EPPlusSoftware/EPPlus#418]
Hi, the following code works fine with Version 5.5

`
using` (var xls = new `ExcelPackage())
{
                var ws = xls.Workbook.Worksheets.Add("Test");
                
                var mergetest = ws.Cells[2, 1, 2, 5];
                mergetest.IsRichText = true;
                mergetest.Merge = true;
                mergetest.Style.WrapText = true;

                var t1 = mergetest.RichText.Add($"Text 1", true);
                t1.Size = 16;
                t1.Bold = true;

                var t2 = mergetest.RichText.Add($"Text 2", true);
                t2.Size = 12;
                t2.Bold = false;

                ws.Row(2).Height = 50;

                xls.SaveAs(new FileInfo(@"c:\temp\test.xlsx"));
}
`

after switching to Version 5.7 the output in Excel is:

System.Object[,]**Text 1**
Text 2

**Comment by JanKallman** [comment]
I'll have a look

**Comment by blindmeis** [comment]
I checked my test code again and it seems that you just need a CellRange and IsRichtext=true to get this wierd output.

```
var mergetest = ws.Cells[2, 1, 2, 2];
mergetest.IsRichText = true;
```

output: System.Object[,]

**Comment by JanKallman** [comment]
Yes, It's related to setting IsRichText on multiple cells. I'll provide a fix for it shortly.
Setting IsRichtext on the first cell will likely work (with a merged range).

**Comment by JanKallman** [comment]
Fixed in version 5.7.1
